src/ParticleDistributions/ParticleDistributions.jl

```diff
@@ -558,69 +669,36 @@ function moments_to_params(dist::AbstractParticleDistribution{FT}, target_moment
   update_params(dist, exp.(r.minimizer))
 end
 
-function moments_to_params(dist::Union{ExponentialAdditiveParticleDistribution, GammaAdditiveParticleDistribution}, target_moments::Array{FT}) where {FT<:Real}
-  # number of equations
-  n_equations = length(target_moments)
+function moments_to_params(dist::Union{ExponentialAdditiveParticleDistribution, GammaAdditiveParticleDistribution, MonodisperseAdditiveParticleDistribution}, target_moments::Array{FT}) where {FT<:Real}
```

Comment:
I wonder if we should rename these as "Mixtures" instead of "Additive Particle Distributions" to help distinguish them from my approach?